src/Transformer.ts

```diff
@@ -63,6 +68,78 @@ export class LuaTransformer {
             case ts.SyntaxKind.InterfaceDeclaration:
                 return this.visitInterfaceDeclaration(node as ts.InterfaceDeclaration);
             // Statements
+            case ts.SyntaxKind.VariableStatement:
+            case ts.SyntaxKind.ExpressionStatement:
+            case ts.SyntaxKind.ReturnStatement:
+            case ts.SyntaxKind.IfStatement:
+            case ts.SyntaxKind.WhileStatement:
+            case ts.SyntaxKind.DoStatement:
+            case ts.SyntaxKind.ForStatement:
+            case ts.SyntaxKind.ForOfStatement:
+            case ts.SyntaxKind.ForInStatement:
+            case ts.SyntaxKind.SwitchStatement:
+            case ts.SyntaxKind.BreakStatement:
+            case ts.SyntaxKind.TryStatement:
+            case ts.SyntaxKind.ThrowStatement:
+            case ts.SyntaxKind.ContinueStatement:
+            case ts.SyntaxKind.EmptyStatement:
```

Comment:
If you want to split like this, is there not some kind of `ts.IsStatement(node)`?

Reply:
There is not, maybe we can reorganise it to look a bit nicer later